### PR TITLE
MB-12019 create move history record for too reweigh request

### PR DESCRIPTION
**Pull request by michaelvaldes-truss**
## Summary

This PR should introduce support for displaying TOO reweigh request events in the move history.
This includes a database migration to add triggers to the reweighs table, an update to the move_history_fetcher to include the reweigh records, and a front end event template to support identifying and displaying reweigh request events.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the UI locally.

```sh
make client_run
```

##### Terminal 2

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the office app
2. Login as a TOO
3. Find a move that has the option for a reweigh request (e.g. locator PENDNG)
4. Navigate to the 'Move task order' tab and click the 'Request reweigh' button located in the 'Shipment weight' section
5. Navigate to the 'Move history' tab and verify that a record is created for the reweigh request and that it is displayed per the story requirements.

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [x] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

## Screenshots
image

### CODE REVIEW

**Review by juliemoon — APPROVED** [review]
This looks good to me! I see it on the Move history page
Screen Shot 2022-04-27 at 2 54 41 PM
.

**Review by reggieriser — APPROVED** [review]
On behalf of the DB reviewers, I reviewed and ran the migration and saw the appropriate triggers on the `reweighs` table.  LGTM!

File: src/constants/moveHistoryEventTemplate.js [hunk 1]
```diff
@@ -157,6 +157,17 @@ export const requestShipmentDiversionEvent = buildMoveHistoryEventTemplate({
...
 });
 
+export const requestShipmentReweighEvent = buildMoveHistoryEventTemplate({
+  action: 'INSERT',
+  eventName: moveHistoryOperations.requestShipmentReweigh,
+  tableName: 'reweighs',
+  detailsType: detailsTypes.PLAIN_TEXT,
+  getEventNameDisplay: () => 'Updated shipment',
+  getDetailsPlainText: (historyRecord) => {
+    return `${shipmentTypes[historyRecord.context[0]?.shipment_type]} shipment, reweigh requested`;
```
Comment (YanZ777) [inline src/constants/moveHistoryEventTemplate.js:167]: **question:** Will the `shipment_type` always be the first or only thing?

Comment (michaelvaldes-truss) [reply src/constants/moveHistoryEventTemplate.js:167]: yep! it's the only thing we get when we build the context

**Review by YanZ777 — APPROVED** [review]
Looks good to me, the question is a non blocker.

**Pull request merged**